resources/charts/kubernetes/values.yaml

```diff
@@ -0,0 +1,43 @@
+apiserver:
+  apiserverKey: 
+  apiserverCert:
+  serviceAccountPub: 
+  caCert: 
+  etcdClientCaCert: 
+  etcdClientCert: 
+  etcdClientKey: 
+  aggregationCaCert: 
+  aggregationClientCert: 
+  aggregationClientKey: 
+  image: k8s.gcr.io/hyperkube:v1.16.3
+  cloudProvider: 
+  etcdServers: 
+  aggregationFlags: 
+  serviceCIDR: 10.0.0.0/24
+  trustedCertsDir: /usr/share/ca-certificates
+controllerManager:
+  serviceAccountKey: 
+  caCert: 
+  caKey: 
+  image: k8s.gcr.io/hyperkube:v1.16.3
```

Comment:
Bootkube now deploy 1.17.0, so chart needs to be updated.

Reply:
updated